pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/SegmentCompletionUtils.java

```diff
@@ -42,6 +43,19 @@ public static String getSegmentNamePrefix(String segmentName) {
   }
 
   public static String generateSegmentFileName(String segmentNameStr) {
```

Comment:
should we call it generateTempSegmentFileName instead since the file name is for a tmp one?